common/src/View/InputEvent.cpp

```diff
@@ -142,21 +143,18 @@ namespace TrenchBroom {
                         m_queue.enqueueEvent(std::make_unique<CancelEvent>());
                         m_dragging = false;
 
-                        // Synthesize a click event if the drag distance was not exceeded.
-                        if (!isDrag(posX, posY)) {
```

Comment:
Why is this isDrag check removed?
We added it in https://github.com/kduske/TrenchBroom/pull/2673 and should keep I think?

Reply:
Yeah, I'll put it back. My thought was that since we are dragging, the drag distance will be exceeded anyway, but in that case we wouldn't have to enqueue a click event at all. I'm not exactly sure why the code looks this way, so I'll leave it as is.